Add links to dotnet runtime repo in docs (#3000)
open-telemetry/opentelemetry-dotnet: docs/metrics/getting-started/README.md

## OpenTelemetry .NET special note

Metrics in OpenTelemetry .NET is a somewhat unique implementation of the

> OpenTelemetry project, as most of the Metrics API are incorporated directly

Reviewer: Change ".NET Core SDK" to ".NET SDK" in line 3?

Author: Thanks. a good change for a next PR.

## OpenTelemetry .NET special note

Metrics in OpenTelemetry .NET is a somewhat unique implementation of the

> OpenTelemetry project, as most of the Metrics API are made part of the [.NET
> runtime](https://github.com/dotnet/runtime) itself. From a high level, what this

Author: ```suggestion
OpenTelemetry project, as most of the Metrics API are offered from the [.NET
runtime](https://github.com/dotnet/runtime) itself. From a high level, what this
```

?

Reviewer 2: Maybe "built into"? I'm not good at English, maybe getting some suggestion from native English speakers?

Reviewer 3: ```suggestion
OpenTelemetry project, as most of the
[Metrics API](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/api.md)
is implemented by the [.NET
runtime](https://github.com/dotnet/runtime) itself. From a high level, what this
```
